third_party/repositories/scala_2_11.bzl

```diff
@@ -516,4 +516,16 @@ artifacts = {
         "artifact": "org.typelevel:kind-projector_%s:0.13.2" % scala_version,
         "sha256": "8f7287973f7f8fc9372b59d36120e3fac5839344f65c8f640351794e8907145c",
     },
+    "io_bazel_rules_scala_scala_interfaces": {
```

Comment:
looks kinda odd, but I couldn't figure out a better solution
the thing is we have to specify all the possible repos in a MODULE.bazel file
there's no access to SCALA_VERSION property from there, so cannot do sth like "use this repo only if scala_version == 3"
I made a doc comment in scala/scala_maven_import_external.bzl repository def which is meant to be used only for bzlmod